### ISSUE TITLE
[TW-1431] Virtual tags cannot be used in color rules

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1458]
_James Sweeny on 2014-10-14T05:25:03Z says:_ 

 Virtual tags cannot be used in color rules. This is easily reproduced in the following .taskrc example:

{code}
color.tag.TODAY=red
{code}

With the above configuration, I would expect tasks due today to be red, but they do not change color, even with all other color rules removed.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-10-14T05:25:03Z
Modified: 2014-10-14T21:53:27Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-10-14T20:31:35Z says:_ 

 I have determined that this is not actually true, and instead, the one virtual tag you chose, is broken.  This is fixed in 2.4.0, but in general, virtual tags, from version 2.3.0 onwards, are available for color rules.  Can you confirm with a different virtual tag?

**Comment by taskwarrior** [comment]
_James Sweeny on 2014-10-14T21:04:42Z says:_ 

 Tested and confirmed working (at least in 1e14097). Not sure which commit was the actual fix, but this is probably safe to close.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-10-14T21:53:27Z says:_ 

 No longer a bug.  The +TODAY behavior was fixed a while ago for 2.4.0, and the color rules work when the virtual tag works.  Confirmed by James - thanks.
